plugins/auth/index.js

```diff
@@ -204,10 +205,26 @@ const authPlugin = {
                             return { isValid: false, credentials: {} };
                         }
 
+                        let scmUser = {};
+
+                        try {
+                            scmUser = await scm.decorateAuthor({
+                                username: user.username,
+                                scmContext: user.scmContext,
+                                token: await user.unsealToken()
+                            });
+                        } catch (e) {
+                            request.log(
+                                ['auth', 'error'],
+                                `Fails to find the user "${user.username}" in ${user.scmContext}.`
+                            );
+                        }
+
                         await createDefaultCollection(user, collectionFactory);
 
                         profile = {
                             username: user.username,
+                            scmUserId: scmUser.id,
```

Comment:
Include the `scmUserId` in the profile same as `scmContext` to get `scmUserId` when authenticating with this JWT.